api/api.go

```diff
@@ -25,9 +25,9 @@ type PayloadProblems struct {
 	PayloadError
 }
 
-// PayloadSubmission represents metadata about a successful submission.
-type PayloadSubmission struct {
-	*Submission
+// PayloadSubmittedIteration represents metadata about a submission of an iteration.
+type PayloadSubmittedIteration struct {
+	*SubmittedIteration
```

Comment:
I don't think there's a time when the CLI will ever handle an iteration that is not Submitted. This should probably be simply `Iteration` along with removing `Submitted` from the other structs